README.md

```diff
@@ -98,7 +100,14 @@ Usage: emqtt_bench conn [--help <help>] [-d <dist>] [-h [<host>]]
   --log_to                   Control where the log output goes. console: 
                              directly to the console      null: quietly, 
                              don't output any logs. [default: console]
-
+  --prometheus               Enable metrics collection via Prometheus.
+                             Usually used with --restapi to enable
+                             scraping endpoint.
+  --restapi                  Enable REST API for monitoring and control. 
+                             For now only serves /metrics. 
+                             Can be set to IP:Port to listen on a specific IP and Port,
+                             or just Port to listen on all interfaces on
+                             that port. [default: disabled]
```

Comment:
```suggestion
  --restapi                  Enable REST API for monitoring and control. 
                             Only takes effect if Prometheus is enabled.
                             For now only serves /metrics. 
                             Can be set to IP:Port to listen on a specific IP and Port,
                             or just Port to listen on all interfaces on
                             that port. [default: disabled]
```
?

Reply:
> Only takes effect if Prometheus is enabled.

I was thinking to add other functionality later, like change config in runtime without restart. Could be useful if you already have 1M connections and don't want to redo them again.